src/browser/browser.zig

```diff
@@ -546,6 +554,9 @@ pub const Page = struct {
     // if no src is present, we evaluate the text source.
     // https://html.spec.whatwg.org/multipage/scripting.html#script-processing-model
     fn evalScript(self: *Page, s: Script) !void {
+        self.current_script = &s;
```

Comment:
Something weird about taking the address of this value and storing it as a field. But it does work, especially since you null it before you exit. 

Would be nice if the module loader took arbitrary state per-load,  but that's a bigger change, and would likely require allocating something on the heap to pass it through.

I do think `evalScript` should take a `*const Script` though. It won't change much - you'll still be storing a stack address in the field - but there's no reason for it to take a copy.